app/wama.web/Views/CheckIn/Waiver.cshtml

```diff
@@ -0,0 +1,80 @@
+﻿@{
+    @using WAMA.Core.Models.DTOs;
+
+    @model WAMA.Core.ViewModel.WaiverViewModel
+    string memberId = (Model?.MemberId) ?? ViewBag.MemberId;
+}
+<div>    
+    <form asp-action="Waiver" asp-antiforgery="true" method="post">        
+        <div class="col-xs-10 col-xs-offset-1">
+            @Html.Partial("DisplayError")
+            <h2>SDSU Climbing Gym Acknowledgement of Risk</h2>
+
+            <h4>
+                PLEASE NOTE: This Waiver of Liability, Release, Acknowledgement of Risk, and Indemnification Agreement (“Waiver Agreement”) is intended
+                to be, and is, legally binding.
+            </h4>
+
```

Comment:
Remember that HTML does not respect newlines.
I believe I mentioned earlier to look into HTML tags to help maintain new lines.

Here's how it looks on my machine.
![image](https://cloud.githubusercontent.com/assets/1667350/23154004/3a1de656-f7d1-11e6-9b52-a88efd7a3288.png)